### PR TITLE
chore: visual refresh on switch component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

Changed the size to 28x18 and updated the radius.
Update the SVG.

before
image

after
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Comment by scomea** [comment]
The selected state seems very "subdued".

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 1]
```diff
@@ -57,214 +51,159 @@ export const switchStyles: (
...
-    :host(:${focusVisible}) .switch {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
+    :host(:${focusVisible}) .switch,
+    :host([aria-checked="true"]:${focusVisible}:enabled) .switch {
+        border-color: ${strokeControlStrongFocus};
+        box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) ${strokeControlStrongFocus};
     }
-
-    .checked-indicator {
+    slot[name="switch"] {
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:97]: Did you test that this works when you _slot_ an item as well? The way this is written it looks like this would slot the default content, but I'm not sure it would catch something slotted.

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 2]
```diff
@@ -57,214 +51,159 @@ export const switchStyles: (
...
-        background: ${foregroundOnAccentActive};
+    .status-message {
+        margin-inline-start: calc(${designUnit} * 2px + 2px);
     }
-
-    :host([aria-checked="true"]:${focusVisible}:not([disabled])) .switch {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
+    :host([aria-checked="true"]) .switch slot[name='switch'],
+    :host([aria-checked="true"]:enabled:hover) .switch slot[name='switch'] {
+        fill: ${neutralForegroundActive};
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:126]: Same question on targeting default slotted content vs. slotted content here - are we sure this catches a scenario when I slot my own SVG?

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 3]
```diff
@@ -46,7 +41,6 @@ export const switchStyles: (
     :host([hidden]) {
         display: none;
     }
-
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:49]: Line 48 below adds margin to the host element - it looks like this has been here, but this should be removed. It should likely be removed in a separate PR from the existing control though. This is problematic as margin relates to space between things which should be applied contextually.

File: packages/web-components/fast-components/src/switch/switch.styles.ts (hunk 1 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/switch/switch.styles.ts:97]: I tested adding a slotted SVG
```html
<fast-switch>
        Custom switch indicator
        <svg
            slot="switch"
            width="8"
            height="8"
            viewBox="0 0 12 12"
            xmlns="http://www.w3.org/2000/svg"
        >
            <path d="M5.28347 1.54556C5.57692 0.95096 6.42479 0.950961 6.71825 1.54556L7.82997 3.79817L10.3159 4.15939C10.9721 4.25474 11.2341 5.06112 10.7592 5.52394L8.96043 7.27736L9.38507 9.75321C9.49716 10.4067 8.81122 10.9051 8.22431 10.5966L6.00086 9.42761L3.7774 10.5966C3.19049 10.9051 2.50455 10.4067 2.61664 9.75321L3.04128 7.27736L1.24246 5.52394C0.767651 5.06111 1.02966 4.25474 1.68584 4.15939L4.17174 3.79817L5.28347 1.54556Z"/>
    </svg>
</fast-switch>
```
image
image

**Review by chrisdholt — APPROVED** [review]

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 4]
```diff
@@ -57,214 +51,159 @@ export const switchStyles: (
...
-    :host([aria-checked="true"]:not([disabled])) .switch:hover .checked-indicator {
-        background: ${foregroundOnAccentHover};
-    }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:active {
-        background: ${accentFillActive};
-        border-color: ${accentFillActive};
+    .label {
+        color: ${neutralForegroundRest};
+        font-size: ${typeRampMinus1FontSize};
```
Comment (bheston) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:116]:
```suggestion
        font-size: ${typeRampBaseFontSize};
```

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 5]
```diff
@@ -57,214 +51,159 @@ export const switchStyles: (
...
+        line-height: ${typeRampMinus1LineHeight};
```
Comment (bheston) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:117]:
```suggestion
        line-height: ${typeRampBaseLineHeight};
```

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 6]
```diff
@@ -57,214 +51,159 @@ export const switchStyles: (
...
+        border-radius: calc(${heightNumber} * 1px);
+        border: calc(${strokeWidth} * 1px) solid ${strokeControlStrongRest};
     }
-
-    .switch:hover {
-        background: ${neutralFillInputHover};
-        border-color: ${neutralStrokeHover};
-        cursor: pointer;
+    :host([aria-checked="true"]) .switch {
+        border-color: ${strokeControlStrongRest};
```
Comment (bheston) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:80]: For improved accessibility I updated the design to have the accent color in selected state for checkbox, radio, and switch.

File: packages/web-components/fast-components/src/switch/switch.styles.ts (hunk 4 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/switch/switch.styles.ts]: updated

File: packages/web-components/fast-components/src/switch/switch.styles.ts (hunk 6 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/switch/switch.styles.ts]: updated to add accent

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 7]
```diff
@@ -57,214 +53,169 @@ export const switchStyles: (
...
              * the checkbox is clicked. Maybe there is a better solution here?
              */ ""
-        } user-select: none;
+        }
+        user-select: none;
+        width: max-content;
     }
-
-    :host([disabled]) {
+    :host(.disabled) {
```
Comment (bheston) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:60]: Maybe I missed something, are we making the change from attributes to class names everywhere? I think attributes are closer to the platform, but I know they don't work in all cases. Is this our standard?

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 8]
```diff
@@ -57,214 +53,169 @@ export const switchStyles: (
...
+    :host(:enabled:active) .switch {
         background: ${neutralFillInputActive};
-        border-color: ${neutralStrokeActive};
+        border-color: ${strokeControlStrongActive};
     }
-
     :host(:${focusVisible}) .switch {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
+        border-color: ${strokeControlStrongFocus};
+        outline: calc(${focusStrokeWidth} * 1px) solid ${strokeControlStrongFocus};
```
Comment (bheston) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:90]: This should be the focus stroke color.

File: packages/web-components/fast-components/src/switch/switch.styles.ts [hunk 9]
```diff
@@ -57,214 +53,169 @@ export const switchStyles: (
...
-    }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:active {
-        background: ${accentFillActive};
-        border-color: ${accentFillActive};
-    }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:active .checked-indicator {
-        background: ${foregroundOnAccentActive};
+    .status-message {
```
Comment (bheston) [inline packages/web-components/fast-components/src/switch/switch.styles.ts:133]: Sync with Jeff's change at https://github.com/microsoft/fluentui/pull/20710/files#diff-d7eaa25d1070145af3a50a2f6ef38364a983d5a57b488909525dc055ae734cbc

**Review by bheston — APPROVED** [review]
